spec/swagger_helper.rb

```diff
@@ -227,6 +241,15 @@
               profile_image: { description: "Profile image (640x640)", type: :string, format: :url },
               profile_image_90: { description: "Profile image (90x90)", type: :string, format: :url }
             }
+          },
+          SharedPodcast: {
+            description: "The podcast the resource belongs to",
```

Comment:
(nitpick/suggestion): 

```suggestion
            description: "The podcast that the resource belongs to",
```